### DISCUSSION TITLE
Adding fixed value to create and update

### DISCUSSION THREAD

**Discussion by Kasparsu** [Laravel-Backpack/community-forum#1001] [Q&A (Help)] [answered]
Was wondering what would be the right way to this?
Trying out Backpack and ran into simple issue. I have `Post` and `User` and there is `User` `hasMany` `Post` relation. Now since I have `CrudController` how would go about setting current `backpack_user()` as `Post` `User`. I mean I can hook model events (https://laravel.com/docs/11.x/eloquent#events) and do it that way, but it seems bit of an hassle. Didn't find from Backpack docs whether there is better way or not.

tried to add `$this->crud->model->user()->associate(backpack_user());` to `setupCreateOperation` and `setup` in my `PostCrudController` backpack just ignores these.

**Comment by Kasparsu** [comment, marked as answer]
Finally found https://backpackforlaravel.com/docs/6.x/crud-operation-create#use-events-in-your-setup-method in the docs. This provides clear solution to the issue
```php
Post::creating(function($entry) {
    $entry->user()->associate(backpack_user());
});
```

**Comment by jcastroa87** [comment]
Im glad you find it.

I will close the issue, but please feel free to re-open or create a new one if needed.

Cheers.
